doc/api/tls.md

```diff
@@ -2,90 +2,145 @@
 
     Stability: 2 - Stable
 
-Use `require('tls')` to access this module.
+The `tls` module provides an implementation of the Transport Layer Security
+(TLS) and Secure Socket Layer (SSL) protocols that is built on top of OpenSSL.
+The module can be accessed using:
 
-The `tls` module uses OpenSSL to provide Transport Layer Security and/or
-Secure Socket Layer: encrypted stream communication.
+```js
+const tls = require('tls');
+```
+
+## TLS/SSL Concepts
 
-TLS/SSL is a public/private key infrastructure. Each client and each
-server must have a private key. A private key is created like this:
+The TLS/SSL is a public/private key infrastructure (PKI). Each client and
+server must have a *private key*.
```

Comment:
If I were a pedant, I'd point out that null ciphers (i.e., no-auth and no-encrypt ciphers) that don't require a key do exist.

Fortunately, I'm not a pedant!